libs/designsystem/src/lib/components/button/button.component.scss

```diff
@@ -99,6 +99,7 @@ $button-width: (
   user-select: none;
   vertical-align: middle;
   white-space: nowrap;
+  position: relative;
```

Comment:
I think the `position: relative` declaration should be moved to the `increase-target-size()` mixin.